Include/pythread.h

```diff
@@ -2,6 +2,12 @@
 #ifndef Py_PYTHREAD_H
 #define Py_PYTHREAD_H
 
+#if !(defined(_POSIX_THREADS) || defined(NT_THREADS))
+#   error "Require native thread feature. See https://bugs.python.org/issue30832"
+#endif
+
+#include <stdbool.h>  /* necessary for TSS key */
```

Comment:
Why is this necessary? Just use an `int` for the initialized flag and avoid the extra dependency - every extra header can be a big deal for non-desktop compilers.

Reply:
Aye, I'd change PyThread_tss_is_created declaration. First of all, I report your proposal to python-dev and python/peps#382.

Reply:
I prefer `bool` myself, but Steve has a point here. There's not much advantage if it means bringing in a header that wouldn't otherwise be necessary, and isn't (currently) used anywhere else in Python.  If we change this it will require a minor update to the PEP.